scalableminds/webknossos
# Add option to invert the color value

Thir PR adds the option to invert the color to the dataset layer settings.

Currently, the histogram is not inverted/mirrored when the color is inverted. Shall I also add this @normanrz?


### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a dataset in view mode, open the dataset settings and toggle the `Invert Color` option on and off. Light areas should turn dark and the other way round.
### Issues:
- fixes #4333

------
